target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -384,6 +384,11 @@ tiny-ac)
 	set_wifi_led "$boardname:orange:wifi"
 	set_usb_led "$boardname:green:usb"
 	;;
+tl-wa801nd-v5)
+	set_wifi_led "$boardname:green:wlan"
+	ucidef_set_led_netdev "lan" "lan" "$boardname:green:lan" "eth0"
+	ucidef_set_led_switch "lock" "lock" "$boardname:orange:lock" "switch0" "0x01"
```

Comment:
This line doesn't make any sense to me. You already have a trigger on the ethernet interface, why to add one ore on switch level?

Reply:
With this device and this line when the ethernet cord is connected, the lock led is off. When the cable is detached, the led become red..

Reply:
OK, correction done